shell/components/SortableTable/index.vue

```diff
@@ -282,6 +284,14 @@ export default {
       default: null,
     },
 
+    /**
+     * The list will always be sorted by these regardless of what the user has selected
+     */
+    mandatorySort: {
```

Comment:
I didn't see an obvious use case for this. Shouldn't we disable sorting in scenarios where this would be used?